values/restund/prod-values.example.yaml

```diff
@@ -0,0 +1,4 @@
+secrets:
+  zrestSecrets:
+    - ""
```

Comment:
it seems `restund` only uses `zrestSecret`, so we can delete the other key from here.
```suggestion
```